vscode/src/copilot/webview/hlsj-qsharp.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Currently Highlight.js for Q# points to https://github.com/fedonman/highlightjs-qsharp/blob/main/src/languages/qsharp.js
+
+export default function (hljs: any) {
+  return {
+    name: "Q#",
+    case_insensitive: true,
+    keywords: {
+      keyword:
+        "namespace open operation function body adjoint controlled self auto if elif else return fail for in new use borrow repeat until fixup",
```

Comment:
This looks a bit out of date: struct, export, import...?

Reply:
I think I took from the original branch without looking 😬  I think you have more context here than I do, I'll ask you later about where we got this from and how exactly we want to consume highlight.js

Reply:
I ended up rewriting the higlight.js language definition. Pretty limited, but it'll do.